### DISCUSSION TITLE
Duda respecto a Tests T0

### DISCUSSION THREAD

**Discussion by RodolfoKompatzki** [IIC2133-PUC/2024-1#43] [Tarea 0] [answered]
Hola! En los tests actualizados hay algunos cuyo nombre termina con "-inc". Estos son los incorrectos? Ya que obtengo errores particularmente en estos. 

Otra duda, como se utiliza el archivo compare.sh para comparar los outputs? Actualmente lo hago con un comparador de texto externo.

**Comment by PabloNavarroY** [comment, marked as answer]
Todos los tests están correctos. Igualmente descarga la ultima versión de estos por si los cambiamos después de que los hayas descargado.

También revisa https://github.com/IIC2133-PUC/2024-1/discussions/5#discussioncomment-8786545
